.github/workflows/docs.yaml

```diff
@@ -42,3 +42,11 @@ jobs:
     with:
       default-landing-page: latest-tag
       additional-unit-test-report-directories: unit-test-report-non-cran
+      deps-installation-method: setup-r-dependencies
+      lookup-refs: |
+        insightsengineering/teal.data
+        insightsengineering/teal.slice
+        insightsengineering/teal.code
+        insightsengineering/teal.logger
+        insightsengineering/teal.reporter
+        insightsengineering/teal.widgets
```

Comment:
GitHub Action for Docs/pkgdown https://github.com/insightsengineering/teal/actions/runs/11159793913